bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -75,7 +76,7 @@ data:
         local -r min_endpoints=$((({{ $replicaCount }} + 1)/2))
 
         for e in "${endpoints_array[@]}"; do
-            if [[ "$e" != "$ETCD_ADVERTISE_CLIENT_URLS" ]] && etcdctl $AUTH_OPTIONS  endpoint health --endpoints="$e"; then
+            if [[ "$e" != "$ETCD_ADVERTISE_CLIENT_URLS" ]] && (unset -v ETCDCTL_ENDPOINTS; exec etcdctl $AUTH_OPTIONS  endpoint health --endpoints="$e"); then
```

Comment:
It is not clear to me why is the `exec` necessary. Can you clarify it?

Reply:
I copied a snippet to do this. I think you're right that the 'exec' is unnecessary as the extra child process isn't a burden and your suggestion simplifies the code. The chart seems to work fine without so I've removed the 'exec'.